portfolio/src/main/webapp/script.js

```diff
@@ -12,35 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+const myName = 'Evan Saracay';
+
+class Typer {
+  constructor() {
+    this.txt = 'Evan Saracay';
+    this.deleting = true;
+    this.speed = 300;
+  }
+  backSpace() {
+    this.txt = myName.substring(0, this.txt.length - 1);
+    this.speed = 100;
+  }
+  type() {
+    this.txt = myName.substring(0, this.txt.length + 1);
+    this.speed = Math.random() * 100 / 2 + 200;
+  }
+}
+
+function txtRotate(text) {
+  if (text.deleting) {
+    text.backSpace();
+  } else {
+    text.type();
+  }
+
+  if (!text.deleting && text.txt === myName) {
+    text.deleting = true;
+  } else if (text.deleting && text.txt === 'Evan') {
+    text.deleting = false;
+  }
+  console.log(text.txt);
+  document.getElementById('intro_content').innerText = text.txt;
+}
+
+
+window.onload =
+    function() {
+  // Function to run on loading which is typing
+  mytyper = new Typer();
+  setInterval(txtRotate, mytyper.speed, mytyper);
 }
 
-function randomQuote() {
-  const quotes =
+/*This function picks a random movie from our list and displays
+it when the user clicks the random movie generator button*/
+function
+randomMovie() {
+  const movies =
       [
-        'Great leaders inspire greatness in others.',
-        'The best confidence builder is experience.',
-        'Sometimes, accepting help is harder than offering it.',
-        'A failure in planning is a plan for failure.',
-        'Never give up hope, no matter how dark things seem.',
-        'Who you were does not have to define who you are.'
+        'Star Wars',
+        'Indiana Jones',
+        'Back to the Future',
+        'Lord of the Rings',
+        'Wall-E',
+        'Robots',
+        'Captain America',
+        'Avengers Infinity War',
+        'Avatar',
+        'The Titanic',
+        'Spider-Man: Into the Spider-Verse',
+        'catch me if you can',
+        'Forrest Gump',
+        'The Aviator',
+        'Inception',
+        'Up',
+        'Gladiator',
+        'A Beautiful Mind',
+        'Interstellar',
+        'The Martian',
+        'The Jungle Book'
       ]
-      // Pick random quote using random for a number 0-1
-      const quote = quotes[Math.floor(Math.random() * quotes.length)];
-  // display in console
-  const container = document.getElementById('starwars-quote');
-  console.log(quote);
-  container.innerText = quote;
+
+      const movie = movies[Math.floor(Math.random() * movies.length)];
```

Comment:
Not sure i'd use const given the value changes

Reply:
Also indentation, but I think const is fine here because the value doesn't change? Maybe the PR was updated since Zach reviewed.

Reply:
I don't think it changed, but this doesn't really fit my intuition of how constants work. That said, I'd defer to Tyler given I don't code in javascript very often..

Reply:
@zmr https://google.github.io/styleguide/jsguide.html#features-use-const-and-let